bin/aws/lib/ssm_tools.sh

```diff
@@ -1,7 +1,18 @@
 #!/usr/bin/env bash
 
+_get_server_instance() {
+    # shellcheck disable=SC2140
+    SERVER_INSTANCE_ID=$(
+        aws ec2 describe-instances \
+            --filters Name=tag:Name,Values="${SERVER_TYPE}" Name="instance-state-name",Values="running" \
+            --query="Reservations[0].Instances[0].InstanceId" \
+            --output=text
+    )
+    echo "${SERVER_INSTANCE_ID}"
```

Comment:
we could just remove the echo and remove the subshell - i tihnk it'll output the same thing - but i'm also not opposed to the over-verbosity of this

Reply:
I'd prefer the verbosity of this for now so that its clear what's going on.